woodwork/logical_types.py

```diff
@@ -256,7 +256,9 @@ def __init__(self, datetime_format=None):
     def transform(self, series):
         """Converts the series data to a formatted datetime. Datetime format will be inferred if datetime_format is None."""
         new_dtype = self._get_valid_dtype(type(series))
-        if new_dtype != str(series.dtype):
+        series_dtype = str(series.dtype)
+
+        if new_dtype != series_dtype and series_dtype not in ["int64"]:
```

Comment:
Can you help me understand what is happening here that required this change? Is there something special with the `int64` dtype that isn't an issue with other integer dtypes?

Reply:
The issue gets raised in `_infer_datetime_format` so we can try to catch that exception there to return None, but haven't found that specific exception object.

Reply:
Hmm...interesting. For inference we are generally (always?) working with a pandas series, but I'm guessing that conversion isn't happening with the code in `_infer_datetime_format`. Could we add a check there for a spark series and convert `first_n` to a pandas series before we guess the format?

Reply:
that could work but there currently is a circular dependency issue because `_is_spark_series` exists in `accessor_utils` which imports the util file containing `_infer_datetime_format`

Reply:
Ugh. Maybe just use an `is_instance` check instead?

Reply:
updated in bb6eceac6d701dced115a617a3a2f66072ec45f0